rnote-cli/src/cli.rs

```diff
@@ -33,13 +33,17 @@ pub(crate) enum Commands {
     },
     /// Exports the Rnote file and saves it in the output file.{n}
     /// The export format is recognized from the file extension of the output file.{n}
+    /// When using --output-format, the same file name is used with the extension changed.{n}
     /// Currently `.svg`, `.xopp` and `.pdf` are supported.
     Export {
         /// the rnote save file
-        rnote_file: PathBuf,
-        /// the export output file
+        rnote_files: Vec<PathBuf>,
+        /// the export output file. Optional if an output format is specified.
         #[arg(short = 'o', long)]
-        output_file: PathBuf,
+        output_file: Option<PathBuf>,
+        /// the export output format. Optional if an output file is specified.
+        #[arg(short = 'f', long)]
+        output_format: Option<String>,
```

Comment:
And it would be best to mention that when this is specified that there can only be one rnote-file